dkpro-core-opennlp-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/opennlp/OpenNlpChunker.java

```diff
@@ -55,6 +58,7 @@
 /**
  * Chunk annotator using OpenNLP.
  */
+@MimeTypeCapability(MimeTypes.APPLICATION_X_OPENNLP_CHUNK)
```

Comment:
Currently, I rely on the `@MimeTypeCapability` in order to find the link between trainer and annotator. If I remove this annotation from the classes, I cannot construct this relation.

Should I replace this annotation with new one, e.g. `@ResourceParameterProducer(MimeTypes.APPLICATION_X_OPENNLP_CHUNK)`?